### PR TITLE
Initial work on hotbar interface

### PR DESCRIPTION
**Pull request by mrpimpunicorn**

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Changes to the hotbar need to be pushed up to the server via [`ChangeHeldItemPacket`](https://github.com/SirCmpwn/TrueCraft/blob/master/TrueCraft.Core/Networking/Packets/ChangeHeldItemPacket.cs). Otherwise this looks fine.

Future:

* Render items
* Change selected item with scroll bar

Can you show me some screenshots of this feature, with several different slots selected? Both TrueCraft and Minecraft textures, please.

File: TrueCraft.Client/TrueCraftGame.cs [hunk 1]
```diff
@@ -38,6 +38,7 @@ public class TrueCraftGame : Game
         public DateTime StartDigging { get; set; }
         public DateTime EndDigging { get; set; }
         public Coordinates3D TargetBlock { get; set; }
+        public int HotbarSelection { get; set; }
```
Comment (SirCmpwn) [inline TrueCraft.Client/TrueCraftGame.cs]: This probably belongs in MinecraftClient as a property that informs the server of the change.

**Comment by mrpimpunicorn** [comment]
flaming hot bars
The top two are from Minecraft's default textures, the bottom two from TrueCraft. Pressing the number keys (1 - 9) selects the appropriate slot. Only the numpad keys are supported by XNA, unfortunately; the alphanumerical keys won't change the hotbar selection.

**Comment by SirCmpwn** [comment]
XNA uses `Keys.D{0-9}`. Screenshots look good.

**Comment by SirCmpwn** [comment]
Ready to merge?

**Comment by mrpimpunicorn** [comment]
Yeah.

**Comment by SirCmpwn** [comment]
Thanks!

**Comment by SirCmpwn** [comment]
FYI - this doesn't look correct when using http://eldpack.com/

**Pull request merged**